handlers/handlers.go

```diff
@@ -134,12 +134,16 @@ func (r RemoveHandler) ShouldHandle() bool {
 
 func (r RemoveHandler) Handle(m *monitors.Monitors) bool {
 	for _, monitor := range *m {
+        if len(monitor.EmptyDesktops()) == 1 {
+            return true
+        }
```

Comment:
Returning `true` signifies that we've made changes to bspwm we should wait to receive the next message before making any more modifications, thus short-circuiting the handler flow.  This will cause all downstream handlers to not get reached

Reply:
Ah, I see that now.

Side note: I will fix formatting in the next commit